packages/vue/src/components/app-header.vue

```diff
@@ -79,8 +83,9 @@ import { DxButton } from 'devextreme-vue/button';
 import { DxToolbar, DxItem } from 'devextreme-vue/toolbar';
 import { useRouter, useRoute } from 'vue-router';
 import { ref } from 'vue';
-import { authInfo as auth, AuthUser } from '../auth';
 
+import ThemeSwitcher from '@/components/theme-switcher.vue';
```

Comment:
looks like you should use relative path as below or use this alias @. I would prefer to use the alias and fix import user-panel. Also looks like we need to group components import. Now we have auth between them